3d/commands.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright 2016 Peter Kropf
```

Comment:
I think for consistency (and scalability) it might make sense to switch to a standardized header for all files in the repo, something like:
"Copyright 2015-2016 Scott Bezek and the splitflap contributors"?

I can go ahead and send a separate diff that will update the existing files, README, LICENSE, etc, to match, and that way it will be clear that the code in this repo is not from a single person. As far as per-file authorship goes, I think git history is probably a reasonable way to preserve attribution since it's automatic and avoids manual churn of header comments as new authors contribute/modify the code. 

I'm largely going off the discussion here http://ben.balter.com/2015/06/03/copyright-notices-for-websites-and-open-source-projects/ but open to alternative suggestions.